db/migrate/20180924213302_rename_rank_to_old_rank_in_answers.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RenameRankToOldRankInAnswers < ActiveRecord::Migration[5.1]
+  def change
+    rename_column(:answers, :rank, :old_rank)
```

Comment:
Good to still keep this around for posterity even though we are phasing it out more.